packages/dappmanager/src/modules/https-portal/index.ts

```diff
@@ -67,6 +68,16 @@ export class HttpsPortal {
 
     // Edit compose to persist the setting
     addNetworkAliasCompose(container, externalNetworkName, aliases);
+
+
+    // Check whether DNP_HTTPS compose has external network persisted
+    const httpsComposePath = ComposeEditor.getComposePath(params.HTTPS_PORTAL_DNPNAME, true)
+    const editor = new ComposeEditor(ComposeEditor.readFrom(httpsComposePath))
+
+    if(editor.getComposeNetwork(externalNetworkName) === null) {
```

Comment:
Since null is a nullish value I would use a nullish check for all possible nullish values
```
if(editor.getComposeNetwork(externalNetworkName) === null)
```

Reply:
Only nullish value `editor.getComposeNetwork` can return is `null`. But, I can change it if improves readability.

Reply:
an empty string is a nullish value as well

Reply:
That is not relevant since:
```
  getComposeNetwork(networkName: string): ComposeNetwork | null {
    return this.compose.networks?.[networkName] ?? null;
  }
  ```